inc/fluent_bit/namespace.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/**
+ * Altis Cloud Fluent Bit logger.
+ *
+ * @package altis/cloud
+ */
+
+namespace Altis\Cloud\Fluent_Bit;
+
+use Altis\Cloud\Fluent_Bit\MsgPackFormatter;
+use Monolog\Handler\SocketHandler;
+use Monolog\Logger;
+
+/**
+ * Check if required constants have been defined
+ *
+ * @return boolean
+ */
+function available() {
+	return defined( 'FLUENT_HOST' ) && defined( 'FLUENT_PORT' );
+}
+
+/**
+ * Retrieve logger for specied $tag_name
+ *
+ * @param string $tag_name Name of the tag to route log messages to
+ * @return Monolog\Logger
+ */
+function get_logger( string $tag_name ) {
+	// TODO how do I need to name the file/class to be autoloaded?
+	require_once __DIR__ . '/class-msgpackformatter.php';
```

Comment:
Typically just include it at the top level, I think the main cloud module file has all the includes IIRC.